docs/deployment.rst

```diff
@@ -56,6 +56,27 @@ whichever service you plan to use) and then running:
 Users can then clone this repository, and point their local `knowledge_repo`
 script at it using :code:`--repo <path_of_cloned_repository>`.
 
+If the git repo is hosted on a remote service like Github, then the knowledge
+repo server will periodically fetch updates, while running as a server. If
+the remote service requires authentication (e.g. a private repository on Github),
+then these requests will fail.
```

Comment:
What do you think of the following rewording of this paragraph:

When serving the posts from this repository using the Knowledge Repo web application, it will attempt to periodically fetch upstream changes to the repository. This can be disabled in the server configuration by setting `INDEXING_UPDATES_REPOSITORIES=False`, in which case you will have to create your own external process for updating the repository in order for the Knowledge Repo to see new posts. If the git repository is served on a remote host that requires authentication, the integrated fetching routines will fail unless you have properly set up your environment with the appropriate authentication.